ospd/command/command.py

```diff
@@ -282,35 +287,49 @@ class GetVts(BaseCommand):
 
     def handle_xml(self, xml: Element) -> str:
         """ Handles <get_vts> command.
+        Writes the vt collection on the stream.
         The <get_vts> element accept two optional arguments.
         vt_id argument receives a single vt id.
         filter argument receives a filter selecting a sub set of vts.
         If both arguments are given, the vts which match with the filter
         are return.
 
-        @return: Response string for <get_vts> command.
+        @return: Response string for <get_vts> command on fail.
         """
+        xml_helper = XmlStringHelper()
 
         vt_id = xml.get('vt_id')
         vt_filter = xml.get('filter')
 
         if vt_id and vt_id not in self._daemon.vts:
             text = "Failed to find vulnerability test '{0}'".format(vt_id)
-            return simple_response_str('get_vts', 404, text)
```

Comment:
I guess we still should fail here? Maybe raise a new OspError (e.g. OspResponseError) which will be catched and converted into a simple_response_str?